src/options/set.rs

```diff
@@ -59,7 +59,7 @@ macro_rules! set_options {
                              &option_names - &expected_option_names));
             }
         }
-	}
+    }
```

Comment:
The changes in this file are not intended, right? (Don't worry about adding more commits to fix, because I usually merge the PRs with squash-and-merge so it will become one commit in the end.)

Reply:
@dandavison 
Oh, I'd like to make sure if this is mistake.

I knew you pointed out in previous PR as well, but I actually didn't get it.
https://github.com/dandavison/delta/pull/321#discussion_r487985240

This bracket is paired to here, starting bracket with 4 spaces, so I thought it was ok.
https://github.com/dandavison/delta/pull/323/files#diff-8b49f2774c51a77ad662f1c88bfd1579R22-R23

The reason why I changed tab into 4 spaces is that whole code looks ruled so. This `macro_rules! builtin_feature`'s argument line is so.
https://github.com/dandavison/delta/blob/master/src/features/mod.rs#L66

Reply:
Ah, OK, I didn't realize I had tabs in there. That's fine then, thanks. (The thing that was making me ask is that usually I like to only have relevant changes in a PR, and do reformatting changes separately. But no need to change it now!)

Reply:
I'm sorry for not separating this. I thought this one is really tiny, but made you confused in fact. I'll be careful.